pkg/component/gardener/apiserver/apiserver_test.go

```diff
@@ -1273,6 +1273,55 @@ kubeConfigFile: /etc/kubernetes/admission-kubeconfigs/validatingadmissionwebhook
 				})
 			})
 
+			Context("ManagedResource reconciliation delay", func() {
+				It("should fail if gardener-apiserver service does not get created from ManagedResource", func() {
```

Comment:
I would name something like:

```suggestion
			Context("when the gardener-apiserver service is not created", func() {
				It("should fail after retrying", func() {
```